tests/test_constants.py

```diff
@@ -357,7 +357,13 @@ def test_serial_comm():
     assert SERIAL_COMM_CHECKSUM_LENGTH_BYTES == 4
     assert SERIAL_COMM_STATUS_CODE_LENGTH_BYTES == 4
     assert SERIAL_COMM_MAX_PACKET_LENGTH_BYTES == 2 ** 16
-    assert SERIAL_COMM_MAX_DATA_LENGTH_BYTES == SERIAL_COMM_MAX_PACKET_LENGTH_BYTES - 14
+    assert SERIAL_COMM_MAX_DATA_LENGTH_BYTES == (
+        SERIAL_COMM_MAX_PACKET_LENGTH_BYTES
+        - SERIAL_COMM_PACKET_INFO_LENGTH_BYTES
+        - SERIAL_COMM_TIMESTAMP_LENGTH_BYTES
+        - SERIAL_COMM_CHECKSUM_LENGTH_BYTES
+        - 10
```

Comment:
note for future: `10` could be set as a constant variable so it's not hanging out by itself here